### PR TITLE
Snapchat API Conversions doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Created Snapchat API Conversions doc

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/advertising/snapchat-conversion.mdx [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+  context: {
+    traits: {
+      email: "alex@example.com",
+      phone: "+1-202-555-0146"
+
+    }
+  }
+})
+```
+
```
Comment (ItsSudip) [inline destinations/advertising/snapchat-conversion.mdx:109]: Put a note here:
We are checking the [eventConversionType](https://marketingapi.snapchat.com/docs/conversion.html#:~:text=CUSTOM_EVENT_3%2C%20CUSTOM_EVENT_4%2C%20CUSTOM_EVENT_5-,event_conversion_type,-Where%20the%20event) from properties. If it is not there we check with the channel and if the channel is also not there we are setting the eventConversionType to OFFLINE.

Code snippet:
rudderanalytics.track("Products Searched", {
query: "HDMI cable",
eventConversionType: "MOBILE_APP"
});

Comment (ameypv-rudder) [reply destinations/advertising/snapchat-conversion.mdx:109]: Added this information.

File: destinations/advertising/snapchat-conversion.mdx [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+RudderStack checks the `eventConversionType`event property to determine the type of event to send to Snap Conversion (web, mobile, or offline). If it is not found, RudderStack checks if `channel` is present. If `channel` is absent too, RudderStack automatically sets `eventConversionType` to `OFFLINE`.
+
+An example highlighting the use of `eventConversionType` is shown below:
+
+```javascript
+
+rudderanalytics.track("Products Searched", {
```
Comment (ameypv-rudder) [inline destinations/advertising/snapchat-conversion.mdx:116]: @ItsSudip should it be `Product Searched` or `Products Searched`?

File: destinations/advertising/snapchat-conversion.mdx [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+
+<table>
+  <tr>
+    <th>RudderStack event</th>
+    <th>Web parameters</th>
+    <th>Mobile app parameters</th>
+    <th>Offline parameters</th>
+  </tr>
+  <tr>
+    <td>Product Searched</td>
```
Comment (ameypv-rudder) [inline destinations/advertising/snapchat-conversion.mdx:134]:
```suggestion
    <td>Products Searched</td>
```

File: destinations/advertising/snapchat-conversion.mdx [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+    </tr>
+</table>
+
+### Event mapping
+
+RudderStack maps the following e-commerce events to the corresponding Snapchat Conversion events:
+
+| **RudderStack event** | **Snapchat Conversion event** |
+|:--------------------------------|:--------------------------|
+| Product Searched | `SEARCH` |
```
Comment (ameypv-rudder) [inline destinations/advertising/snapchat-conversion.mdx:216]:
```suggestion
| Products Searched | `SEARCH` |
```

File: destinations/advertising/snapchat-conversion.mdx (hunk 2 above)
Comment (ItsSudip) [reply destinations/advertising/snapchat-conversion.mdx:116]: It is Products Searched. You can follow [here](https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-ecommerce-events-specification/browsing/#products-searched).

**Review by ItsSudip — APPROVED** [review]

File: destinations/advertising/snapchat-conversion.mdx [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+</div>
+
+- **Pixel ID**: Enter your Snap Pixel ID. This is required for sending the web and offline events.
+
+<div class="infoBlock">
+
+For more information on getting your Snap Pixel ID, refer to the <a href="#faq">FAQ</a> section below.
+</div>
+
+- **Snap App ID**: Enter your Snapchat App ID.
```
Comment (ujjwal-ab) [inline destinations/advertising/snapchat-conversion.mdx:62]: Snap App Id is also required for app events, we can update this also.

**Pull request closed without merging**